rustls/src/server/handy.rs

```diff
@@ -1,6 +1,8 @@
 use alloc::sync::Arc;
 use alloc::vec::Vec;
 use core::fmt::Debug;
+use pki_types::CertificateDer;
+use std::vec;
```

Comment:
nit: the `pki_types` import should be on its own, separated by whitespace from the `alloc`/`core` imports above it and the `crate` imports below it.

I also think `std::vec` should probably be `alloc::vec` for no-std?

Reply:
Is it correct as it's now?